### ISSUE TITLE
How to get this: "select count(*) from ABC" ?

### ISSUE THREAD

**Issue by Rainer-Lang** [Raizlabs/DBFlow#96]
Select.count(Abc.class, Condition.column(Abc$Table.THIS_IS_MY_PRIMARY_KEY).isNotNull());

I'm curious if this is the "best" or easiest solution.

**Comment by agrosner** [comment]
You can also do: ```new Select().count().from(table)
.where(Condition.column(Abc$Table.THIS_IS_MY_PRIMARY_KEY).isNotNull()).count()
```

What's nice is that you have options. The method you specified is just a wrapper around that statement to make it even easier to run the query.

**Comment by damson** [comment]
Are you sure about the last `count()` ?  Doesn't it gonna count the
`select().count()` result ? Because if it does, it will always return 1 for
counting always a number, equivalent to one result.
Im not sure about what's happening in the `Where.count()` method, Im just
intrigued.


On Tue, Mar 17, 2015 at 1:36 PM, Andrew G <notifications@github.com> wrote:

> Closed #96 <https://github.com/Raizlabs/DBFlow/issues/96>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Raizlabs/DBFlow/issues/96#event-256804934>.
>



-- 
*DAGNET Damien*
